src/services/services.ts

```diff
@@ -1807,6 +1807,15 @@ namespace ts {
             return range && createTextSpanFromRange(range);
         }
 
+        function getSpanForPosition(fileName: string, position: number): TextSpan {
+            synchronizeHostData();

```

Comment:
>synchronizeHostData(); [](start = 12, length = 22)

Why synchronize?

Reply:
No longer valid as the method has been removed.

Reply:
I still find the synchronization surprising, but I see that the existing GTD code path does it.